docs/advanced-topics/linear-probe.md

```diff
@@ -0,0 +1,61 @@
+(linear-probe)=
+# {octicon}`pin` Linear Probe
+
+## Why freezing?
+
+Depending on your task and the amount of training data,
+sometimes it is unnecessary to tune the entire model.
+In these cases,
+freezing the weights of the pre-trained model and fine-tuning specific layers is sufficient enough.
+Besides, freezing can reduce the training time considerably.
+
+Finetuner allows you to fine-tune a Linear Probe (or Projection Head) easily.
```

Comment:
Is there a wikipedia page or paper for this? I don't know what that is.

Reply:
different ppl use different words, for example: https://arxiv.org/pdf/2002.05709.pdf

Reply:
updated everything to projection head